get-artifacts.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+here=$(realpath "$0")
+here=$(dirname "$here")
+cd "$here"
+SOLR_VERSION="7.7.2"
+cloud_config_version="2021.03"
```

Comment:
why is cloud_config_version in here?

Reply:
removed it. It came by mistake while copying from cloudconfig repo.